packages/commonwealth/server/routes/status.ts

```diff
@@ -45,6 +45,30 @@ const status = async (
       models.ChainCategoryType.findAll(),
     ]);
 
+    const chainsWithSnapshots = await Promise.all(
+      chains.map(async (chain) => {
+        const snapshot_spaces = await models.CommunitySnapshotSpaces.findAll({
+          where: {
+            chain_id: chain.id,
+          },
+          include: {
+            model: models.SnapshotSpace,
+            as: 'snapshot_space',
+          },
+        });
+
+        const snapshot_space_names = snapshot_spaces.map((space) => {
+          return (space as any).snapshot_space?.snapshot_space;
```

Comment:
This was because we used include in the previous findAll to attach the snapshot_space to the query, and it was giving me a type error if I tried to access snapshot_space without the cast. Is there a better way?

Reply:
what was the type error?

Reply:
that snapshot_space was not a field on space

Reply:
I mean -- this sounds like an interfaces definition problem we should solve, rather than something to hack around with `any`. In general, we shouldn't use `any` unless we have a very specific reason.

Reply:
🫡